docker/compose/live-reload.override.yml

```diff
@@ -0,0 +1,36 @@
+---
+version: '2.2'
+services:
+  meteor:
+    build:
+      context: ../../
+      dockerfile: docker/compose/mozdef_meteor/Dockerfile
+    restart: always
+    # command: bash -c 'cd /opt/mozdef/envs/mozdef/meteor && meteor npm install --save-dev && rm -rf .meteor/local && /usr/bin/meteor'
```

Comment:
Can we delete the commented out line here or is there some value in keeping it around? I've got commented out lines in other docker-compose files to make it easy to expose ports for example, so if it's not leftover cruft, we can keep it around.